akka-persistence-typed/src/main/scala/akka/persistence/typed/state/internal/DurableStateStoreInteractions.scala

```diff
@@ -58,13 +58,13 @@ private[akka] trait DurableStateStoreInteractions[C, S] {
 
   protected def internalDelete(
       ctx: ActorContext[InternalProtocol],
-      cmd: Any,
+      @unused cmd: Any,
       state: Running.RunningState[S]): Running.RunningState[S] = {
 
     val newRunningState = state.nextRevision().copy(state = setup.emptyState)
     val persistenceId = setup.persistenceId.id
 
-    onDeleteInitiated(ctx, cmd)
+    // TODO Might need to call hook method for Telemetry
```

Comment:
I approved just before you commit these TODOs. 

Are you planning to investigate it further or is this PR ready for merging?

Reply:
I would like to leave that for later. Better to add the right hooks when actually implementing the instrumentation. These hooks were premature in the first place.